certs/certs.go

```diff
@@ -64,6 +66,7 @@ func WriteNewCerts(certFile, keyFile string, server bool) error {
 		KeyUsage:              x509.KeyUsageKeyEncipherment | x509.KeyUsageDigitalSignature | x509.KeyUsageCertSign,
 		BasicConstraintsValid: true,
 		DNSNames:              []string{"localhost"},
+		IPAddresses:           []net.IP{net.IPv4(127, 0, 0, 1)},
```

Comment:
Sure. As I wrote, this is not perfect... If `config.Host` is supposed to be an IP address, the `DNSNames` is wrong, too.